app/Models/Cart.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Cart extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'user_id',
+        'name',
+        'phone',
+        'address',
+    ];
+
+    public function user()
+    {
+        return $this->belongsTo(User::class,'cart_id', 'user_id');
```

Comment:
```suggestion
        return $this->belongsTo(User::class);
```
thế này thôi là được rồi e ơi